proto/cosmos/distribution/v1beta1/distribution.proto

```diff
@@ -115,12 +115,17 @@ message ValidatorSlashEvents {
 }
 
 // FeePool is the global fee pool for distribution.
+// It holds decimal coins. Once whole those coins can be burned or distributed to the community pool.
 message FeePool {
   repeated cosmos.base.v1beta1.DecCoin community_pool = 1 [
     (gogoproto.nullable)     = false,
     (amino.dont_omitempty)   = true,
-    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins"
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins",
+    deprecated               = true
   ];
+
+  repeated cosmos.base.v1beta1.DecCoin decimal_pool = 2
```

Comment:
This is the same as the community pool in term on type, but renaming so that there is no confusion

Reply:
what is decimal pool suppose to signify?

Reply:
That it gets the decimal coins, the scrums. This concept does not need to be known outside the module, but keeping the community pool as said above would have been confusing.